dockers/docker-bgp/template_list

```diff
@@ -0,0 +1,4 @@
+bgpd.conf.j2  /etc/quagga/bgpd.conf    quagga:quaggavty  0644
+zebra.conf.j2 /etc/quagga/zebra.conf   root:root         0644
+isolate.j2    /usr/sbin/bgp-isolate    root:root         0755
+unisolate.j2  /usr/sbin/bgp-unisolate  root:root         0755
```

Comment:
>nisolate.j2 [](start = 1, length = 11)

suggest implement this feature as:
1. make fake config files in Dockerfile, set the correct ownership, groupship and mode. I guess the vanilla ones are already good.
2. copy file and maintain the target file attribute. ref: [http://serverfault.com/questions/273949/cp-not-want-to-overwrite-permissions/273956](http://serverfault.com/questions/273949/cp-not-want-to-overwrite-permissions/273956)